app/src/app/data.service.ts

```diff
@@ -18,15 +23,23 @@ export class DataService {
   public splittedTraining: string[];
   public keyDetails: KeyDetails;
   public timelineRole: TimelineRole;
-
+  public capability: Capability;
+  public band: Band;
+  public family: Family;
+  public roles: Roles;
   public carouselRole: CarouselRole[] = [];
+
   constructor(private http: HttpClient) {
-    this.getUser();
-    this.getRoleSpecification();
+    this.getBand();
```

Comment:
these should be call at the point at which they are required

Reply:
This will be changed in #36